server/store/store.go

```diff
@@ -11,11 +13,12 @@ type Store interface {
 // PollStore allows the access polls in the store.
 type PollStore interface {
 	Get(id string) (*poll.Poll, error)
-	Save(poll *poll.Poll) error
-	Delete(poll *poll.Poll) error
+	Insert(*poll.Poll) error
+	Update(prev *poll.Poll, new *poll.Poll) error
+	Delete(*poll.Poll) error
```

Comment:
I thought about if a `prev` variable is needed for `Delete`, but in the end every time you do delete, you want to do it anyway.